typings/index.d.ts

```diff
@@ -0,0 +1,106 @@
+declare module "discord-starboards" {
+    import { EventEmitter } from "events";
+    import * as DJS from "discord.js";
+  
+    export type TranslateClick = (msg: DJS.Message) => void;
+  
+    export interface StarboardOptions {
+      storage?: string | boolean;
+      translateClickHere?: string | TranslateClick;
+    }
+  
+    export interface Starboard {
+      channelID: string;
+      guildID: string;
+      options: StarboardDefaultCreateOptions;
+    }
+  
+    export interface ColorOptions {
+      colors: string[];
```

Comment:
Maybe rather colors: (string|number|number[])[] (I don't know if it's possible)

-> https://discord.js.org/#/docs/main/stable/typedef/ColorResolvable

Reply:
Yes, it's possible. I can simply use `DJS.ColorResolvable`, I'll update this when home